config/locales/en.yml

```diff
@@ -43,6 +43,11 @@ en:
       delimiter: ","
       separator: "."
 
+  activerecord:
```

Comment:
Can you elaborate on why the activerecord attribute is needed in thelacales files?

Reply:
To get the validation error "Race/Ethnicity can't be blank". The field in the table is called 'demographic', and by default, the "validates :demographic, presence: true" validation will give "Demographic can't be blank".
That is a way to name the field in the error message like it's labeled in the form.

Reply:
That's pretty use of the locale file